src/minter-contract-utils/src/operation_store.rs

```diff
@@ -352,6 +376,28 @@ mod tests {
         }
     }
 
+    #[test]
+    fn should_get_page_for_operations() {
```

Comment:
Add additional tests for all combinations of offset and count (e.g. Some/Some, Some/None, None/Some, None/None)